bs-react-native-next/src/apis/NetInfo.md

```diff
@@ -0,0 +1,43 @@
+---
+id: apis/NetInfo
+title: NetInfo
+wip: true
+---
+
+```reason
+type info = {
+  .
+  "_type": NetInfoType.t,
+  "effectiveType": NetInfoEffectiveType.t,
```

Comment:
Can we name the modules `ConnectionType` and `EffectiveConnectionType` to match the RN docs?

The actual file/module names can be `NetInfo_ConnectionType` and `NetInfo_EffectiveConnectionType`, and then we can do:

```re
module ConnectionType = NetInfo_ConnectionType;
module EffectiveConnectionType = NetInfo_EffectiveConnectionType;
```

Reply:
I doubt we would have name collision if we simply renamed the modules `ConnectionType` and `EffectiveConnectionType`, would that be alright?

Reply:
IMHO they should be in the NetInfo scope (accessible as `NetInfo.ConnectionType` and `NetInfo.EffectiveConnectionType`). If you want to have them in separate files, I think we should follow the convention I described above, same as I did for `WebView` and related types.